web/components/lotteryCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Card, Typography, colors } from '@mui/material';
+import { Lottery } from '../types';
+import { Done, Sync } from '@mui/icons-material';
+
+interface Props {
+  lottery: Lottery;
+  selected: boolean;
+  onSelect: () => void;
+}
+
+const selectedStyles = {
+  boxShadow: `inset 0 0 0 2px ${colors.blue[400]}`,
+};
+
+const disabledStyles = {
+  backgroundColor: colors.grey[100],
+};
+
+export default function LotteryCard({ lottery, selected, onSelect }: Props) {
```

Comment:
While naming the file from which the React component is exported please follow the Pascal case naming convention: `LotteryCard.tsx`